.github/workflows/prepare-release.yml

```diff
@@ -2,6 +2,11 @@ name: Prepare Release
 
 on:
   workflow_dispatch:
+    inputs:
+      version:
```

Comment:
How do we pass this argument `version`. Is it required when we manually trigger the workflow?

Reply:
We pass it in the github UI when we manually trigger the workflow - it's not required and if you leave it blank, it will use the version from the changelog.